test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Form.html

```diff
@@ -7,8 +7,8 @@
 
     <h2>Form Tag Helper Test</h2>
     
-    <form></form>
-    <form action="/HtmlGeneration_Home/Form" method="post"></form>
+    <form><input name="__RequestVerificationToken" type="hidden" value="{0}" /></form>
+    <form action="/HtmlGeneration_Home/Form" method="post"><input name="__RequestVerificationToken" type="hidden" value="{0}" /></form>
```

Comment:
Amazed we only have two `<form>` elements without attributes we recognized. That's sad.